.github/workflows/github-ci.yaml

```diff
@@ -137,6 +126,9 @@ jobs:
       run: |
         echo `ruff --version`
         ruff .
+    - name: Test with mypy
+      run : |
+        mypy pypdf
```

Comment:
Does mypy behave the same for different Python versions?

Reply:
i simply checked py3.8 ~ py3.11, they behaved the same. but for py3.6, probably it's not.

Reply:
Lets see if I get good answers for this via https://stackoverflow.com/q/77219925/562769

I suspect that it should give the same results, so I tend to agree with this change.